### PR TITLE
chore(release): 2.0.0-beta.1 release candidate

### PR DESCRIPTION
**Pull request by rbarilani**
@cjhowald @gabrielhl @gregdeane @dami-gg @sambhav-gore  What about releasing a new version... we need the new nice features and updates.

Please check the changelog because I think that we are missing to document some breaking changes..

### CODE REVIEW

**Comment by sambhav-gore** [comment]
@rbarilani Couple of points :

- I think we should change the README and remove modernizr requirement (I believe after removing touch/no-touch) this is not a requirement anymore.
- Shouldn't this PR be for master branch ?

File: dist/css/dress-code.css [hunk 1]
```diff
@@ -1824,8 +1660,8 @@ body {
 
 @font-face {
   font-family: "DressCodeIcons";
-  src: url("../fonts/DressCodeIcons.eot");
-  src: url("../fonts/DressCodeIcons.eot?#iefix") format("eot"), url("../fonts/DressCodeIcons.woff") format("woff"), url("../fonts/DressCodeIcons.ttf") format("truetype"), url("../fonts/DressCodeIcons.svg#DressCodeIcons") format("svg");
+  src: url("../fonts/DressCodeIcons.eot?v=2.0.0-beta");
+  src: url("../fonts/DressCodeIcons.eot?#iefix") format("eot"), url("../fonts/DressCodeIcons.woff?v=2.0.0-beta") format("woff"), url("../fonts/DressCodeIcons.ttf?v=2.0.0-beta") format("truetype"), url("../fonts/DressCodeIcons.svg?v=2.0.0-beta#DressCodeIcons") format("svg");
 }
```
Comment (sambhav-gore) [inline dist/css/dress-code.css]: The versions here should be updated. Can we automate this somehow ?

**Comment by rbarilani** [comment]
I opened it against develop to be able to change what is needed before releasing....
@gregdeane I would suggest to open a branch here `` release/2.0.0-beta.1``` so I can close this one and reopen a pull request against that branch..... so we can work to fine tune the release... WDYT?

File: dist/css/dress-code.css (hunk 1 above)
Comment (rbarilani) [inline dist/css/dress-code.css]: @sambhav-gore Sorry my fault yes it's automated I miss the order of tasks for releasing.... good catch!

**Comment by gregdeane** [comment]
@rbarilani I don't think i follow. What are you suggesting?

**Comment by rbarilani** [comment]
@gregdeane I am suggesting to open a release branch so I can open a PR against that branch, and then we can continue working on release fine tuning....

**Comment by gregdeane** [comment]
@rbarilani Have a meeting set up next week with SC team to discuss further. Thx!

**Comment by rbarilani** [comment]
I close this one, since the things are moving directly to 2.0.0. Thanks guys.

**Pull request closed without merging**